src/test/unit/views/subscription-management.test.ts

```diff
@@ -26,6 +29,20 @@ const expectedCourtRowsCount = 6;
 
 let htmlRes: Document;
 
+const rawData = fs.readFileSync(path.resolve(__dirname, '../../../main/resources/mocks/userSubscriptions.json'), 'utf-8');
+const subscriptionsData = JSON.parse(rawData);
+sinon.stub(SubscriptionRequests.prototype, 'getUserSubscriptions').returns(subscriptionsData.results[0]);
+
+jest.mock('axios', () => {
```

Comment:
you dont need to mock axios, match the format weve been using and mock the service like the rest of the tests

Reply:
axios mock removed